File: rmt/src/test/scala/de/flower/rmt/ui/manager/page/venues/VenueEditPageTest.scala
package de.flower.rmt.ui.manager.page.venues

import de.flower.rmt.test.AbstractWicketIntegrationTests
import org.testng.annotations.Test
/**
 * 
 * @author flowerrrr
 */

class VenueEditPageTest extends AbstractWicketIntegrationTests {

    @Test
    def testRender() {
        wicketTester.startPage(new VenueEditPage())
        wicketTester.dumpPage()
    }


}

File: rmt/src/test/java/de/flower/rmt/ui/page/venues/manager/VenueEditPageTest.java
package de.flower.rmt.ui.page.venues.manager;

import de.flower.rmt.test.AbstractRMTWicketIntegrationTests;
import org.testng.annotations.Test;



public class VenueEditPageTest extends AbstractRMTWicketIntegrationTests {

    @Test
    public void testRender() {
        wicketTester.startPage(new VenueEditPage());
        wicketTester.dumpPage();
    }
}